packages/rancher-aks-operator/rancher-aks-operator/package.yaml

```diff
@@ -1,3 +1,3 @@
-url: https://github.com/rancher/aks-operator/releases/download/v1.2.0-rc1/rancher-aks-operator-1.2.0-rc1.tgz
+url: https://github.com/rancher/aks-operator/releases/download/v1.2.0-rc2/rancher-aks-operator-1.2.0-rc2.tgz
 version: 103.0.0
 doNotRelease: false
```

Comment:
this needs a removal, https://github.com/rancher/charts/pull/2898#discussion_r1303351251?